R/graph.curve.params.R

```diff
@@ -8,7 +8,11 @@
 #'
 #' @examples
 #' \dontrun{
-#' plot1 <- graph.curve.params(curve_params)
+#' curve <-
+#'   typhoid_curves_nostrat_100 |>
+#'   filter(antigen_iso %in% c("HlyE_IgA", "HlyE_IgG"))
```

Comment:
```suggestion
#'   dplyr::filter(antigen_iso %in% c("HlyE_IgA", "HlyE_IgG"))
```

should fix the issue you were getting, so we  can remove `\dontrun`